bin/node/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aleph-node"
-version = "0.6.0"
```

Comment:
Actually, don't we also want ot update runtime and other crates?

Reply:
runtime updated, other crates not related imo